ci-operator/step-registry/install-operators/OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
```

Comment:
Before we get too far into PR's being opened/merged, lets solve the approves/reviewers now (my personal opinion).

We just need to reach out to prob `forum-dp-platform` and get admin of the `openshift` org to create a new team and provide admin access to certain people. Then those admins can control who is added.

There are a couple approaches we can go here regarding teams created:

- **Very Simple**: One Team (e.g. `cspi-maintainers`) which would can be used for the `approvers` and `reviewers` roles.
- **Somewhat Simple**: Two teams (e.g. `cspi-approvers` & `cspi-reviewers`) which then would be assigned to their correct roles `approvers || reviewers`. These teams then can apply to any type of jobs (e.g. layered product interop testing, service integration testing, etc). Everyone is in the loop.
- **Starting To Get Complicated**: Two teams (e.g. `cspi-lp-maintainers` && `cspi-<X>-maintainers`) which then would have people in different teams based on projects. Then each team would be part of the `approvers` and `reviewers`.
- **Lets Get Complicated!**: Four teams (e.g. `cspi-lp-approvers`, `cspi-lp-reviewers`, `cspi-<X>-approvers`, `cspi-<X>-reviewers`. Based on above just being more granular around who can do what to a PR.

This can kick off the discussion prior to submitting  any request for team(s). Doing this now just will reduce the amount of future PR's needed to change everyone's users if needed.

Reply:
Since this is a more of a nice to have and not a blocker, we can ignore this for now. After some further research, using teams within OWNER files is not supported by prow. There is other ways to create aliases that can be used. This can be explored at a later time.

https://docs.prow.k8s.io/docs/components/plugins/approve/approvers/